cmd/_integration-tests/transport/http_test.go

```diff
@@ -400,6 +400,67 @@ func TestStrangeRPCName(t *testing.T) {
 	}
 }
 
+func TestHTTPErrorStatusCodeAndNilHeaders(t *testing.T) {
+	// See handlers/handlers.go for implementation
+	// Returns status code http.StatusTeapot
+	req, err := http.NewRequest("GET", httpAddr+"/status/code", strings.NewReader(""))
```

Comment:
Yes, wow I was literally imagining there was some body there; but there isn't :sheep: but yeah, those bodies can be `nil` and it would be a tad clearer